vhost-user-backend/src/backend.rs

```diff
@@ -87,6 +89,18 @@ pub trait VhostUserBackend: Send + Sync {
     /// function.
     fn set_backend_req_fd(&self, _backend: Backend) {}
 
+    /// This function gets an owned file descriptor that the front-end can use otherwise
```

Comment:
`the front-end can use otherwise no file descritptor`, What would you like to say here? It sounds like a cut sentence.